src/content/docs/change-tracking/change-tracking-webhooks.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Notify your team about deployments'
+tags:
+  - APM
+metaDescription: "After you set up the changes you want to monitor, you can use a webhook to notify your colleagues about the impacts of those changes."
+---
+
+
+import webHookTest from 'images/tracking_screenshot-crop_webhook_test.webp'
+
+
+
+After you record a deployment for an APM application entity, whether it's with the change tracking feature or the older [REST API](/docs/apm/apm-ui-pages/events/record-deployments/#api-instructions), you can notify your team members using a webhook.
+
+## Get your webhook destination URL [#get-webhook-url]
+
+You can send deployment data to a variety of webhook destinations. Follow the instructions for the tool you're using to get the webhook URL.
+
+If you're using Slack, use the legacy New Relic alerts app:
+
+1. Log in to your Slack account as an admin, and then go to **Apps**.
+1. Search for **New Relic Alerts**, and click on that tile when it appears.
+1. In the **New Relic Alerts** listing, click the **Configuration** button below the New Relic icon.
+1. Click the **Configuration** tab below the heading **New Relic Alerts**.
+1. On the **Configuration** tab, click the pencil icon.
+1. Scroll down to the **Webhook URL** section, and click **Copy URL**.
+
+Once you have the URL, complete the steps in the next section to configure webhook notifications.
+
+## Configure webhook notifications for deployments [#configure-webhook]
+
+Insert the webhook URL in the New Relic UI:
+
+1. Go to the deployment notifications configuration screen: **[one.newrelic.com](https://one.newrelic.com/) > (user menu) > Administration > Integrations > Deploy notifications**.
+1. Paste your webhook URL into the **Webhook URL** field and click **Save**.
+1. Click **Send a test request** to send an example payload with artificial data to your webhook URL.
+    <img
+      style={{align: "left"}}
```

Comment:
I'm assuming that this is already in dark mode? But the UI is just white?

Reply:
@homelessbirds the screenshot I received is in light mode instead of the preferred dark mode. I didn't want to ask for another shot. Did you think this was unreadable with the white background?

Reply:
@rhetoric101 I don't think it's unreadable. It's fine with me if you merge it with this.